goblin/driver/cluster.py

```diff
@@ -165,8 +166,15 @@ def _get_message_serializer(self, config):
             config['message_serializer'] = my_import(message_serializer)
         return config
 
-    def config_from_module(self, filename):
-        raise NotImplementedError
+    def config_from_module(self, module):
+        if isinstance(module, str):
+            module = importlib.import_module(module)
+        config = dict()
+        for item in dir(module):
+            if not item.startswith('_') and item.lower() in self.DEFAULT_CONFIG:
```

Comment:
Just curious, why do you check `not item.startswith('_')`? Can something start with `_` and be in `self.DEFAULT_CONFIG`?